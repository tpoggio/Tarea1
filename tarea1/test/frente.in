encolar 15 4.0
frente
encolar 95 24.0
frente
encolar 85 2.0
encolar 65 3.0
encolar 25 67.0
encolar 35 -12.0
encolar 75 2.5
encolar 45 2.1
encolar 55 -24.0
encolar 85 32.0
frente
encolar 10 10.0
frente
Fin