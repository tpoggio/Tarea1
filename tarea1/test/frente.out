1>Encolar. 
2>(15,4.00)
3>Encolar. 
4>(15,4.00)
5>Encolar. 
6>Encolar. 
7>Encolar. 
8>Encolar. 
9>Encolar. 
10>Encolar. 
11>Encolar. 
12>Encolar. 
13>(15,4.00)
14>Encolar. 
15>(15,4.00)
16>Fin.
